src/IssueViz.UnitTests/Editor/Common/TaggerTestHelper.cs

```diff
@@ -34,6 +35,9 @@ internal static class TaggerTestHelper
         public static ITextSnapshot CreateSnapshotAndBuffer(int length = 999) =>
             CreateBufferMockWithSnapshot(length).Object.CurrentSnapshot;
 
+        public static ITextBuffer CreateBufferWithSnapshot(int length = 999) =>
```

Comment:
Convenience method - sometimes the caller is more interested in the buffer, sometimes the snapshot.

Reply:
The method names are really confusing, `CreateSnapshotAndBuffer` vs `CreateBufferWithSnapshot`. 
Maybe just CreateSnapshot and CreateBuffer?